skillmap/src/App.tsx

```diff
@@ -432,6 +435,14 @@ class AppImpl extends React.Component<AppProps, AppState> {
         }
     }
 
+    protected focusCurrentActivity = () => {
+        const node = document.querySelector("[data-activity=" + this.props.activityId + "] button");
+        (node as HTMLElement | SVGElement).focus();
```

Comment:
can node be null? or is this pretty safe

Reply:
nope, this should never be null